tests/phpunit/tests/rest-api/rest-posts-controller.php

```diff
@@ -121,7 +121,9 @@ public function wpSetUpBeforeRequest( $result, $server, $request ) {
 	}
 
 	public function save_posts_clauses( $orderby, $query ) {
```

Comment:
I'm unclear why this needs to change.

"Fixing" the tests always makes me nervous because it can often mean there's an unintended side effect of the new code. I once fixed the tests while at ⬛ and broke every new post on ⬛com.au, ⬛com.au and ⬛com.au -- it was not a great day.

Reply:
@peterwilsoncc 13 tests fail in rest post controller test when not using this _fix_. 
Previous functionality _wp_get_post_revisions()_ uses _get_children()_ that uses _suppress_filters -> true_ by default. As this new query does not use _supress_filters_, it will blow the posts_clauses count.

Reply:
> As this new query does not use _supress_filters_, it will blow the posts_clauses count used in other tests.

Thanks.

I'd suggest setting the default arguments for this new function to be identical to the existing query: a merge of the `get_revisions`, `get_children` and `get_posts` defaults. Only the pagination & `no_found_rows` settings need to change.

That will ensure that the count and most recent revision returned by this function matches the result of the existing function.